docs/user_guides/fs/feature_view/query.md

```diff
@@ -157,3 +157,80 @@ The filters can be applied at any point of the query:
         .join(locationFg.selectAll(), Seq("location_id"), Seq("id"), "left")
         .filter(rainFg.getFeature("location_id").eq(10)))
     ```
+
+#### Joins and/or Filters on feature view query
+
+Exiting feature view query can be extended with new joins and/or new filters. However, this operation
+will not update feature view metadata and persist it with new query.
+
+=== "Python"
+    ```python
+    fs = ...
+    wind_speed_fg = fs.get_feature_group(name="wind_speed_fg", version=1)
+    rain_fg = fs.get_feature_group(name="rain_fg", version=1)
+    feature_view = fs.get_feature_view(“rain_dataset”, version=1)
+    feature_view.query\
+    .join(wind_speed_fg.select_all())\
+    .filter((rain_fg.location_id == 54)
+    ```
+
+=== "Scala"
+    ```scala
+    val fs = ...
+    val windSpeedFg = fs.getFeatureGroup("wind_speed_fg", 1)
+    val rainFg = fs.getFeatureGroup("rain_fg", 1)
+    val featureView = fs.getFeatureView(“rain_dataset”, 1)
+    featureView.getQuery().
+        join(windSpeedFg.selectAll()).
+        filter(rainFg.getFeature("location_id").eq(54))
```

Comment:
Not sure if this works in jupyter, but it would be much better to read.
```suggestion
    featureView.getQuery()
        .join(windSpeedFg.selectAll())
        .filter(rainFg.getFeature("location_id").eq(54))
```